apps/www/content/docs/components/01.autocomplete.md

```
@@ -128,6 +118,19 @@ name: AutocompleteAsyncDemo
 ---
 ::
 
+## API Reference
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)